code/game/objects/effects/decals/turfdecal/flooring_decals.dm

```diff
@@ -110,6 +110,12 @@
 	}													\
 	/obj/effect/turf_decal/spline/fancy/corner/##color_name {	\
 		icon_state = "spline_fancy_corner"				\
+	}													\
+	/obj/effect/turf_decal/road/line/##color_name {		\
+		color = ##tile_color							\
+	}													\
+	/obj/effect/turf_decal/road/line/edge/##color_name {		\
+		color = ##tile_color							\
```

Comment:
holy damn that formatting is beyond awful. The formatting was already there so its whatever; but whoever started it should be taken outback

Reply:
hi, here and waiting to be taken out behind the shed

although seriously that's just how defines gotta be, or rather, it's the most readable way of having them

Reply:
🔫

Reply:
Rest in peace mark